cli/dcoscli/data/help/node.txt

```diff
@@ -25,7 +25,7 @@ Commands:
         cluster.
     diagnostics create
         Create a diagnostics bundle. Nodes can be: ip address, hostname, mesos ID
-        or key words "all", "masters", "agents".
+        or key words "all", "masters", "agents" (notice the quotes around the keywords).
```

Comment:
We need to make the same change here: https://github.com/dcos/dcos-cli/blob/master/cli/tests/data/help/node.txt#L26-L28